report_xlsx/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
```

Comment:
be consistent and remove the copyright notice in all init files

Reply:
You can remove also `coding` line

Reply:
@elicoidal Suggested changes have been included.

Reply:
@pedrobaeza coding could be removed from all files, isn't it?

Reply:
Yeah, exactly. See second point of https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration